api/api-base/src/main/java/com/thoughtworks/go/api/ApiController.java

```diff
@@ -34,6 +35,7 @@
 
 public abstract class ApiController implements ControllerMethods, SparkController {
     private static final Set<String> UPDATE_HTTP_METHODS = new HashSet<>(Arrays.asList("PUT", "POST", "PATCH"));
+    private static final String MULTIPART_REQ_ATTR = "org.eclipse.jetty.multipartConfig";
```

Comment:
Can we push this to somewhere closer to Jetty? My worry is, if a Jetty upgrade changes this property, we won't know to change it here.

Reply:
I will move it into `GoServer`, as I don't want to add any more module dependencies (mostly because I'm unsure of consequences). This might be close enough to the `jetty9` module?

Reply:
Better yet, I found the constant in `org.eclipse.jetty.server.Request.__MULTIPART_CONFIG_ELEMENT` and extracted this into the `server` module as `RequestUtils.configureMultipart()`.